app/assets/javascripts/select-kit/addon/components/select-kit/select-kit-row.js

```diff
@@ -115,9 +114,13 @@ export default Component.extend(UtilsMixin, {
     return this.getValue(this.selectKit.highlighted);
   }),
 
-  isHighlighted: propertyEqual("rowValue", "highlightedValue"),
+  isHighlighted: computed("rowValue", "highlightedValue", function () {
+    return this.rowValue === this.highlightedValue;
+  }),
 
-  isSelected: propertyEqual("rowValue", "value"),
+  isSelected: computed("rowValue", "value", function () {
```

Comment:
import { propertyEqual } from "discourse/lib/computed"; is not defined in wizard